### DISCUSSION TITLE
visible pilot and copilot

### DISCUSSION THREAD

**Discussion by KaiBVG** [flybywiresim/aircraft#3485] [General]
just saw that the feature visibile pilot and copilot was added. In the CDU-misc I see both options, but not at LSK left and right like in the screenshot but at LSK left 1 and 2. Switching them but nothing happens. Is this a feature for the future already or do I have to download some avatars? My simsettings are at avatar - standard. 
A second questions: is the pilot/copilot-feature implemented in this: https://flybywiresim-packages.nyc3.cdn.digitaloceanspaces.com/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip WIP too or do I have to decide using one of them?

**Comment by hotshotp** [comment]
It has been reverted because it caused some issues
